### DISCUSSION TITLE
[Improvement] HDR support

### DISCUSSION THREAD

**Discussion by fox-john** [Kong/insomnia#3726] [Ideas]
I have same bug for VS Code, look here : https://github.com/microsoft/vscode/issues/68069

In insomnia, screen is very black with HDR enabled.
Left = HDR screen, right = no HDR screen
![screen very black](https://i.imgur.com/H3LYVj1.jpg)

PS: Sorry for my bad english.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by gschier** [comment]
From the issue you liked, it looks like this may be a [workaround](https://github.com/microsoft/vscode/issues/68069#issuecomment-472994404) for now:

```
insomnia --force-color-profile srgb
```

**Comment by fox-john** [comment]
@gschier thx ! It's work for now ! Great job :) !
